docs/sources/flow/reference/components/discovery.triton.md

```diff
@@ -0,0 +1,125 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/discovery.triton/
+title: discovery.triton
+---
+
+# discovery.triton
+
+`discovery.triton` discovers [Triton][] Container Monitors and exposes them as targets.
+
+[Triton]: https://www.tritondatacenter.com
+
+## Usage
+
+```river
+discovery.triton "LABEL" {
+	account    = ACCOUNT
+	dns_suffix = DNS_SUFFIX
+	endpoint   = ENDPOINT
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name               | Type           | Description                                                         | Default       | Required
+------------------ | -------------- | ------------------------------------------------------------------- | ------------- | --------
+`account`          | `string`       | The account to use for discovering new targets.                     |               | yes
+`role`             | `string`       | The type of targets to discover.                                    | `"container"` | no
+`dns_suffix`       | `string`       | The DNS suffix which should be applied to target.                   |               | yes
+`endpoint`         | `string`       | The Triton discovery endpoint (e.g. 'cmon.us-east-3b.triton.zone'). |               | yes
```

Comment:
```suggestion
`endpoint`         | `string`       | The Triton discovery endpoint (e.g. 'cmon.us-east-3b.triton.zone'). |               | yes
```

Should we include examples in the argument descriptions? This has come up before and we've generally opted for no examples here. If we do include the example, use `for example` instead of `e.g.`.

Reply:
example removed